components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/ClientServices.cs

```diff
@@ -423,222 +408,6 @@ public class ClientServices : IClientHost, IEndpointServices, IEndpointDiscovery
             }
         }
 
-        /// <summary>
-        /// Create application configuration for client
-        /// </summary>
-        /// <returns></returns>
-        private ApplicationConfiguration CreateApplicationConfiguration(
-            TimeSpan operationTimeout, TimeSpan sessionTimeout) {
-
-            // mitigation for bug in .NET Core 2.1
-            var effectiveAppCertStoreType = _configuration.AppCertStoreType;
```

Comment:
Is this logic still present?

Reply:
after deeper analyses, I concluded this workaround is not necessary.